src/main/java/org/broadinstitute/hellbender/tools/copynumber/allelic/alleliccount/AllelicCountCollector.java

```diff
@@ -1,29 +1,19 @@
 package org.broadinstitute.hellbender.tools.copynumber.allelic.alleliccount;
 
-import htsjdk.samtools.*;
-import htsjdk.samtools.filter.DuplicateReadFilter;
-import htsjdk.samtools.filter.NotPrimaryAlignmentFilter;
-import htsjdk.samtools.filter.SamRecordFilter;
-import htsjdk.samtools.reference.ReferenceSequenceFileWalker;
-import htsjdk.samtools.util.IntervalList;
-import htsjdk.samtools.util.SamLocusIterator;
+import htsjdk.samtools.util.Locatable;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.utils.Nucleotide;
 import org.broadinstitute.hellbender.utils.SimpleInterval;
 import org.broadinstitute.hellbender.utils.Utils;
-import org.broadinstitute.hellbender.utils.io.IOUtils;
-import org.broadinstitute.hellbender.utils.param.ParamUtils;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
 
-import java.io.File;
-import java.io.IOException;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 /**
- * Collects reference/alternate allele counts at sites.  The alt count is defined as the total count minus the ref count,
+ * Collects reference/alternate allele counts at a site.  The alt count is defined as the total count minus the ref count,
```

Comment:
I would've just left this doc as is, since the method `collectAtLocus` collects counts at a single site, but the collector can (and will most likely) be used to collect counts at multiple sites.  Perhaps "at specified sites"? @